static/parse-hosting/index.html

```diff
@@ -95,34 +97,39 @@
       display: none;
     }
 
-    @media (min-width: 540px) {
-      body {
-        background: #1976d2
-      }
+    .parse-box {
+      background: #fff;
+      padding: 40px 10px;
+      max-width: 540px;
+    }
 
-      .wrapper {
-        display: flex;
-        justify-content: center;
-        align-items: center;
-      }
+    .parse-box__header {
+      text-align: center;
+      margin-bottom: 40px;
+    }
 
+    .parse-box__header img {
+      margin: 0 auto 40px;
+    }
+
+    @media (min-width: 540px) {
       .parse-box {
         padding: 60px 70px;
         border-radius: 4px;
         box-shadow: 0 0 50px rgba(0, 0, 0, .37);
       }
-
-      .form input,
-      .form__message {
-        margin-bottom: 20px;
-      }
     }
     
     @media (max-width: 768px) {
```

Comment:
please keep it mobile first. and why changing RWD behaviour which is already coded from 540px?